package.json

```diff
@@ -14,14 +14,16 @@
     "flow-typed": "rimraf flow-typed/npm && flow-typed install --overwrite || true",
     "lint": "cross-env NODE_ENV=development eslint --cache --format=node_modules/eslint-formatter-pretty .",
     "lint-fix": "yarn --silent lint --fix; exit 0",
+    "lint-json-fix": "prettier --ignore-path .eslintignore --parser json --write '**/*(*.json|.*(babelrc|eslintrc|stylelintrc))'",
```

Comment:
Why did we add a new script for this instead of reuse an existing script?

Reply:
Add to lint-fix?

Reply:
Could make postlint-fix into this, but it would be long.
`"postlint-fix": "concurrently \"prettier --ignore-path .eslintignore --single-quote --write '**/*.*(js|jsx)'\" \"prettier --ignore-path .eslintignore --parser json --write '**/*(*.json|.*(babelrc|eslintrc|stylelintrc))'\""`

Reply:
Could make postlint-fix into this, but it would be a bit long.
`concurrently \"prettier --ignore-path .eslintignore --single-quote --write '**/*.*(js|jsx)'\" \"prettier --ignore-path .eslintignore --parser json --write '**/*(*.json|.*(babelrc|eslintrc|stylelintrc))'\"`

Reply:
One of the issues with the boilerplate is that is has way too many npm scripts. Try combining this with the `lint-fix` script

Reply:
Merged into `postlint-fix`. This required `.prettierrc` to be added back to avoid the `--parser json` flag (since it would cause errors for js/jsx).

The problem with doing this though, is that if a user wanted to add another filetype to be included in the lint they need to include it in both .prettierrc and package.json.

Reply:
Also used the more commonly used glob patterns, since it reduces the length of the command. To be consistent, I also had to change it in the lint-staged config and `postlint-styles-fix`.

(i.e. `*.*(js|jsx)` becomes `*.{js,jsx}`)